nexus/examples/config-file.toml

```diff
@@ -40,6 +40,10 @@ level = "info"
 #mode = "stderr-terminal"
 
 # Example output to a file, appending if it already exists.
-mode = "file"
-path = "logs/server.log"
-if_exists = "append"
+#mode = "file"
```

Comment:
This file was supposed to be exactly the same as `nexus/examples/config.toml`, except it logs to a file instead of stdout.  I think maybe you didn't mean to comment out this section?